### PR TITLE
MOBILE-2681 Readme and pull request template update

### PR DESCRIPTION
**Pull request by jamesromo-wf**
Description
---
Need to update readme and pull request template based on new repo changes

What Was Changed
---
-  Readme and pull request template update

Acceptance Criteria
---
- Readme and pull request template make sense for new PRs

Merge Checklist
---
- [x] Author of commit is not a reviewer
- [ ] CR +1
- [x] +10 (includes code review and pulling in the code/testing)
- [x] bundle exec pod lib lint WMobileKit.podspec --allow-warnings passes
- [x] Security review if applicable
- [x] Bump the WMobileKit.podspec, Source/Info.plist, and Example/WMobileKitExample/Info.plist
using ```version_bump.sh```according to semantic versioning.

Example: (5.1.0 is the old version 5.1.1 is the new version)
```ruby
./version_bump.sh 5.1.0 5.1.1
```

---
Please Review: @Workiva/mobile

### CODE REVIEW

**Comment by aviary2-wf** [comment]
## Raven
<details>
<summary>Number of Findings: 0</summary>

</details>

File: README.md [hunk 1]
```diff
@@ -3,8 +3,9 @@
 [![Version](https://img.shields.io/cocoapods/v/WMobileKit.svg?style=flat)](http://cocoapods.org/pods/WMobileKit)
 [![License](https://img.shields.io/cocoapods/l/WMobileKit.svg?style=flat)](http://cocoapods.org/pods/WMobileKit)
 [![Platform](https://img.shields.io/cocoapods/p/WMobileKit.svg?style=flat)](http://cocoapods.org/pods/WMobileKit)
+[![BuddyBuild](https://dashboard.buddybuild.com/api/statusImage?appID=59d266080dd7d400015d1a6f&branch=master&build=latest)](https://dashboard.buddybuild.com/apps/59d266080dd7d400015d1a6f/build/latest?branch=master)
 
-WMobileKit is a Swift library containing various custom UI components to provide functionality outside of the default libraries. Each component is customizable and featured in a sample app to demonstrate its functinality!
+WMobileKit is a Swift library containing various custom UI components to provide functionality outside of the default libraries. Each component is customizable and featured in a sample app to demonstrate its functionality!
```
Comment (jordanross-wf) [inline README.md]: 🥇

**Comment by jordanross-wf** [comment]
+10 to documentation and template changes, example app runs as expected still

**Comment by jamesromo-wf** [comment]
@Workiva/release-management-pp

**Pull request merged**